app/src/main/java/com/mapbox/maps/testapp/examples/DebugModeActivity.kt

```diff
@@ -56,35 +60,8 @@ class DebugModeActivity : AppCompatActivity() {
     scaleBarPlugin = mapView.scalebar
     scaleBarPlugin.enabled = false
     scaleBarPlugin.textColor = ContextCompat.getColor(this@DebugModeActivity, R.color.primary)
-    displayOnSecondDisplayButton.setOnClickListener {
-      scaleBarPlugin.enabled = true
-
-      mapboxMap.setDebug(debugOptions, false)
-      debugOptions.clear()
-
-      if (checkboxTileBorders.isChecked) {
-        debugOptions.add(MapDebugOptions.TILE_BORDERS)
-      }
-      if (checkboxCollision.isChecked) {
-        debugOptions.add(MapDebugOptions.COLLISION)
-      }
-      if (checkboxDepthBuffer.isChecked) {
-        debugOptions.add(MapDebugOptions.DEPTH_BUFFER)
-      }
-      if (checkboxOverdraw.isChecked) {
-        debugOptions.add(MapDebugOptions.OVERDRAW)
-      }
-      if (checkboxParseStatus.isChecked) {
-        debugOptions.add(MapDebugOptions.PARSE_STATUS)
-      }
-      if (checkboxStencilClip.isChecked) {
-        debugOptions.add(MapDebugOptions.STENCIL_CLIP)
-      }
-      if (checkboxTimestamps.isChecked) {
-        debugOptions.add(MapDebugOptions.TIMESTAMPS)
-      }
-      mapboxMap.setDebug(debugOptions, true)
-    }
+    scaleBarPlugin.enabled = true
```

Comment:
The scale bar is disabled firstly and enabled while clicking. IMO we can remove codes related to scalebar now.